code/modules/fish/fish_items.dm

```diff
@@ -0,0 +1,218 @@
+
+var/global/list/fish_items_list = list("goldfish" = /obj/item/weapon/fish/goldfish,
+									"clownfish" = /obj/item/weapon/bananapeel/clownfish,
+									"shark" = /obj/item/weapon/fish/shark,
+									//"baby space carp" = /obj/item/weapon/fish/babycarp,
+									"catfish" = /obj/item/weapon/fish/catfish,
+									"feederfish" = /obj/item/weapon/reagent_containers/food/snacks/feederfish,
+									"salmon" = /obj/item/weapon/fish/salmon,
+									"shrimp" = /obj/item/weapon/reagent_containers/food/snacks/shrimp,
+									"electric eel" = /obj/item/weapon/fish/electric_eel,
+									"glofish" = /obj/item/weapon/fish/glofish
+,
+									)
+
+//////////////////////////////////////////////
+//			Aquarium Supplies				//
+//////////////////////////////////////////////
+
+/obj/item/weapon/fishtools/fish_egg_scoop
+	name = "fish egg scoop"
+	desc = "A small scoop to collect fish eggs with."
+	icon = 'icons/obj/fish_items.dmi'
+	icon_state = "egg_scoop"
+	slot_flags = SLOT_BELT
+	throwforce = 0
+	w_class = 2
+	throw_speed = 3
+	throw_range = 7
+
+/obj/item/weapon/fishtools/fish_net
+	name = "fish net"
+	desc = "A tiny net to capture fish with. It's a death sentence!"
+	icon = 'icons/obj/fish_items.dmi'
+	icon_state = "net"
+	slot_flags = SLOT_BELT
+	throwforce = 0
+	w_class = 2
+	throw_speed = 3
+	throw_range = 7
+
+	suicide_act(mob/user)			//"A tiny net is a death sentence: it's a net and it's tiny!" https://www.youtube.com/watch?v=FCI9Y4VGCVw
```

Comment:
You're gonna get an angry comic here if you do this.

Do /obj/item/weapon/fishtools/fish_net/suicide_act(mob/user) instead